pkgs/development/python-modules/pyside6/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, cmake
+, ninja
+, qt6
+, python
+, shiboken6
+, libxcrypt
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pyside6";
+
+  inherit (shiboken6) version src;
+
+  postPatch = ''
+    # Don't ignore optional Qt modules
+    substituteInPlace sources/pyside6/cmake/PySideHelpers.cmake \
+      --replace \
+        'string(FIND "''${_module_dir}" "''${_core_abs_dir}" found_basepath)' \
+        'set (found_basepath 0)'
+
+    cd sources/${pname}
```

Comment:
Probably could be done by `sourceRoot = "sources/${pname}"`.

Reply:
Just tested it, won't work because inside the archive there's a subdirectory called `pyside-setup-everywhere-src-6.5`

Reply:
So `sourceRoot = "pyside-setup-everywhere-src-${lib.versions.majorMinor version}/sources/${pname}";` should work.

Reply:
Updated :+1: